chasm/lib/scheduler/scheduler_idle_tasks_test.go

```diff
@@ -0,0 +1,156 @@
+package scheduler_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+	"go.temporal.io/server/chasm"
+	"go.temporal.io/server/chasm/lib/scheduler"
+	"go.temporal.io/server/chasm/lib/scheduler/gen/schedulerpb/v1"
+	"google.golang.org/protobuf/types/known/durationpb"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type idleTasksSuite struct {
+	schedulerSuite
+	executor *scheduler.SchedulerIdleTaskExecutor
+}
+
+func TestIdleTasksSuite(t *testing.T) {
+	suite.Run(t, &idleTasksSuite{})
+}
+
+func (s *idleTasksSuite) SetupTest() {
+	s.SetupSuite()
```

Comment:
Why do we need to set up the suite on every test? If this is supposed to be used for test setup, the method should be called `SetupTest`.